### PR TITLE
Support Ruby 2.0+

### PR DESCRIPTION
**Pull request by JuanitoFatas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Support Ruby 2.1+ [PR#10](https://github.com/choonkeat/active_waiter/pull/10)
```
Comment (JuanitoFatas) [inline CHANGELOG.md]: Can we remove `PR` prefix from `PR#10`, `#10` is good enough? @winston

**Comment by choonkeat** [comment]
Why are we abandoning a ruby version for test helper code?

**Comment by JuanitoFatas** [comment]
> Why are we abandoning a ruby version for test helper code?

We are using a syntax that is not supported in Ruby 2.0.0: https://travis-ci.org/choonkeat/active_waiter/jobs/64030311.

**Comment by choonkeat** [comment]
> We are using a syntax that is not supported in Ruby 2.0.0

we're using that syntax in a test convenience helper. that's not important

**Comment by JuanitoFatas** [comment]
> we're using that syntax in a test convenience helper. that's not important

You are right. Let me fix that.

**Comment by JuanitoFatas** [comment]
Closed for now. There will be a PR covers this.

**Pull request closed without merging**